src/main/java/io/odpf/firehose/config/ObjectStorageSinkConfig.java

```diff
@@ -49,6 +50,10 @@
     @DefaultValue("dt=")
     String getTimePartitioningDatePrefix();
 
+    @Key("SINK_OBJECT_STORAGE_FAIL_ON_DESERIALIZATION_ERROR")
```

Comment:
We don't need this. Looks very odd in the constructor. By default we can return those messages...Maybe DLQ writer can decide what to do with the type of error.

Reply:
removed this